contrib/fuzzing/FuzzDaemon.dict

```diff
@@ -0,0 +1,85 @@
+"0"
+"7"
+","
+":"
+"2.1e24"
+
+"true"
+"false"
+"null"
+
+"\"\""
+"\"\":"
+
+"{}"
+",{}"
+":{}"
+"{\"\":0}"
+"{{}}"
+
+"[]"
+",[]"
+":[]"
+"[0]"
+"[[]]"
+
+"''"
+"\\"
+"\\b"
+"\\f"
+"\\n"
+"\\r"
+"\\t"
+"\\u0000"
+"\\x00"
+"\\0"
+"\\uD800\\uDC00"
+"\\uDBFF\\uDFFF"
+
+"\"\":0"
+"//"
+"/**/"
+
+
+# Things like geojson, json-ld, ...
+"$ref"
+"type"
+"coordinates"
+"@context"
+"@id"
+"@type"
+
+# Strings with truncated special values
+"{\"foo\":fa"
+"{\"foo\":t"
+"{\"foo\":nul"
+
+"{"
+"}"
+"\"qty\": 1, \"qty\": -1"
+"\"qty\": 1, \"qty\\ud800\": -1"
+"\"qty\": 1, \"qt\\y\": -1"
+"/*"
+"*/"
+"\""
+"1.7976931348623157e+308"
+"5e-324"
+"9007199254740991"
+"-9007199254740991"
+
+"}="
+
+",,"
+"{\"\":"
+
+# Specific for repositories file
+"Repositories"
+"busybox"
+"latest"
+"sha256"
+"91e54dfb11794fad694460162bf0cb0a4fa710cfa3f60979c177d920813e267c"
+"jess/hollywood"
+"registry"
+"5000"
+"HEAD"
+"master"
```

Comment:
nit: looks like this file is missing a trailing newline; if adding one doesn't break how it's used, perhaps you could add?